www/network.js

```diff
@@ -40,24 +40,10 @@ function NetworkConnection() {
 
 /**
  * Get connection info
- *
- * @param {Function} successCallback The function to call when the Connection data is available
- * @param {Function} errorCallback The function to call when there is an error getting the Connection data. (OPTIONAL)
  */
-NetworkConnection.prototype.getInfo = function(successCallback, errorCallback) {
-    exec(successCallback, errorCallback, "NetworkStatus", "getConnectionInfo", []);
-};
-
-var me = new NetworkConnection();
-var timerId = null;
-var timeout = 500;
-
-channel.createSticky('onCordovaConnectionReady');
-channel.waitForInitialization('onCordovaConnectionReady');
-
-channel.onCordovaReady.subscribe(function() {
-    me.getInfo(function(info) {
-        me.type = info;
+NetworkConnection.prototype.getInfo = function() {
```

Comment:
jak już edytujemy to proponuję pokusić się o zmianę nazwy, np. checkCurrentStatus. W appce wołana metoda wygląda jakby gdzieś miała coś zwracać